### PR TITLE
Assigment 2b

### PR DESCRIPTION
**Pull request by poll2000**

### CODE REVIEW

File: Assignment2/poll2000-fromJSONtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            }
+    ]
+}
+
+{
+    "@context": {
+        "pab": "http://www.pablo.es/schema/",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "fullName": "rdfs:label",
+        "hometown": "rdfs:label",
```
Comment (ocorcho) [inline Assignment2/poll2000-fromJSONtoJSON-LD.jsonld:26]: These cannot be rdfs:label. The same with other properties

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
Assignment 2b has not been completed.
Now @dgarijo to check the assignment 4

File: Assignment4/poll2000-200386/task06.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1VtTXH84ug6QfFKtGO5QcqisMG99MVk4A
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/poll2000-200386/task06.py]: This won't work in python. It's notebook magic

File: Assignment4/poll2000-200386/task06.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+g.add((ns.Smith, vcard.Family, Literal("Smith")))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+# Visualize the results
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, vcard.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/poll2000-200386/task06.py]: not a property in vcard.

File: Assignment4/poll2000-200386/task07.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+# TO DO
+# Visualize the results
+
+#RDFLib
+print("RDFLib:")
+ns = Namespace("http://somewhere#")
+for sub, per, obj in g.triples((None, RDFS.subClassOf, ns.Person)):
+  print(sub, "es subclase de Person.")
+  for sub2, per2, obj2 in g.triples((None, RDFS.subClassOf, sub)):
+    print(sub2, "is subclass of ", sub, "which in turn is of Person type.")
```
Comment (dgarijo) [inline Assignment4/poll2000-200386/task07.py]: second loop not needed

File: Assignment4/poll2000-200386/task07.py [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+
+#SPARQL
+print("SPARQL:")
+from rdflib.plugins.sparql import prepareQuery
+myQuery = prepareQuery('''
+  SELECT DISTINCT ?x
+  WHERE{
+	  ?x (rdfs:subClassOf/rdfs:subClassOf*) ns:Person
```
Comment (dgarijo) [inline Assignment4/poll2000-200386/task07.py]: not needed here. with subclassOf is enough

**Review by dgarijo — CHANGES_REQUESTED** [review]
Small changes from tasks 6 and 7

**Review by dgarijo — APPROVED** [review]

**Comment by dgarijo** [comment]
@ocorcho assignment 4 is ok. Shall I merge even though there are changes requested by you?

**Pull request merged**